src/balances/balances.controller.spec.ts

```diff
@@ -308,11 +308,39 @@ describe('Balances Controller (Unit)', () => {
 
         await request(app.getHttpServer())
           .get(`/chains/${chainId}/safes/${safeAddress}/balances/usd`)
-          .expect(503)
-          .expect({ message: 'Service unavailable', code: 503 });
+          .expect(HttpStatus.SERVICE_UNAVAILABLE)
+          .expect({
+            message: 'Service unavailable',
+            code: HttpStatus.SERVICE_UNAVAILABLE,
+          });
 
         expect(mockNetworkService.get.mock.calls.length).toBe(2);
       });
     });
   });
+
+  describe('GET /balances/', () => {
+    it('Success', async () => {
+      const fiatCodesResult = fiatCodesResultFactory();
+      mockNetworkService.get.mockResolvedValueOnce({ data: fiatCodesResult });
+
+      await request(app.getHttpServer())
+        .get('/balances/supported-fiat-codes')
+        .expect(HttpStatus.OK)
+        .expect(['USD', 'EUR', 'AED', 'AFN', 'ALL']);
```

Comment:
nitpick: if you change the `fiatCodesResultFactory` this test ends up failing so it seems that you are testing what the factory returns.

I'd use the factory if you don't care about its internal data (just that it has "correct" data). In order to test specific data I'd do that setup in the test itself.